### PR TITLE
feat(Crunchyroll): add series page support and multiples fix

### LINKED ISSUE #6635: PreMiD not working with new crunchyroll Beta
### Presence name

Crunchyroll

### Description

PreMiD works correctly with www.crunchyroll.com but does not work with https://beta.crunchyroll.com/

When using the beta version, the activity of crunchyroll pops up on discord but is stuck on browsing. Does not show what show you are watching, episode, timestamps, etc. All US users are forced to watch on the beta site and cannot watch from the regular site unless they are logged out of their crunchyroll account.

Thank you!

### Alternatives

_No response_

### Additional context

_No response_

### LINKED ISSUE #6662: PreMid
### Steps to Reproduce

https://premid.app/

### Expected Behavior

My Crunchyroll to display with episode/time

### Actual Behavior

Only showed I was browsing as long as crunchyroll was open

### Page URL

The discord app status display

### Screenshots

image

### Operating System

Windows 10

### Browser and Version

Google

### Additional Comments

No clue why this started happening but it also happens both google browser and firefox browser
2 weeks-1 month ago this 1st occured

### PR DESCRIPTION
**Pull request by RisingSunLight42**
### Changelog

- Remove useless await in front of `strings` variable
- Resolves #6635 (and probably Resolves #6662) by correcting wrong querySelectors.
- Fix the whole manga part by correcting querySelectors.
- Add support for the "series" page when you're viewing an anime page for example on the beta version of the website.
- Change if statement for the showCover setting, now at the end of the file.
- Add multiple location variable like href and pathname to avoid repetition.
- Remove useless checks
- Simplification of some code parts

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

crunchyroll_proof_1

crunchyroll_proof_6

crunchyroll_proof_5

crunchyroll_proof_4

crunchyroll_proof_3

crunchyroll_proof_2

</details>

### CODE REVIEW

**Review by EncryptedDev — APPROVED** [review]

File: websites/C/Crunchyroll/presence.ts [hunk 1]
```diff
@@ -180,15 +163,31 @@ presence.on("UpdateData", async () => {
...
 				},
 			];
-			presence.setActivity(presenceData, !paused);
 		}
+	} else if (pathname.includes("/series")) {
+		presenceData.details = strings.viewPage;
+		presenceData.state = document.querySelector("h1.title").textContent;
+		presenceData.largeImageKey =
+			document.querySelector<HTMLMetaElement>("[property='og:image']")
+				?.content ?? "lg";
```
Comment (Bas950) [inline websites/C/Crunchyroll/presence.ts:180]: maybe just set it to an imgur link instead of lg. For future maintainability

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/Crunchyroll/presence.ts (hunk 1 above)
Comment (RisingSunLight42) [reply websites/C/Crunchyroll/presence.ts:180]: 👍 I'll change that when I can

Comment (RisingSunLight42) [reply websites/C/Crunchyroll/presence.ts]: Resolved in https://github.com/PreMiD/Presences/pull/6641/commits/e33d6786dfb5f4f5383997f4349c6afdb3842ccf

**Comment by RisingSunLight42** [comment]
Also will probably resolve #6662

**Review by Bas950 — APPROVED** [review]

**Pull request merged**